samcli/commands/pipeline/bootstrap/cli.py

```diff
@@ -204,6 +205,15 @@ def do_cli(
             config_dir=PIPELINE_CONFIG_DIR, filename=PIPELINE_CONFIG_FILENAME, cmd_names=_get_command_names()
         )
 
+        click.secho(
+            Colored().green(
+                "\nThe ARNs of created resources have been written to "
+                f"{os.path.join(PIPELINE_CONFIG_DIR, PIPELINE_CONFIG_FILENAME)}.\n"
+                f"It will be used next time you run `sam pipeline bootstrap` or "
+                f"`sam pipeline init` commands in this directory."
```

Comment:
Maybe a bit more succinct? Backticks make it already pretty clear they're commands.

```suggestion
                f"`sam pipeline init` in this directory."
```